doc/reference/trd105-time.md

```diff
@@ -0,0 +1,455 @@
+Kernel Time HIL
+========================================
+
+**TRD:** 105<br/>
+**Working Group:** Kernel<br/>
+**Type:** Documentary<br/>
+**Status:** Draft <br/>
+**Obsoletes:** 101 <br/>
+**Author:** Guillaume Endignoux, Amit Levy, Philip Levis, and Jett Rink <br/>
+**Draft-Created:** 2021/07/32 <br/>
```

Comment:
I don't think this month as 32 days. :smile:

Reply:
Good catch. Fixed. Thanks!